spec/features/movies/detail_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+RSpec.describe "When I visit the movies detail page", type: :feature do
+  before :each do
+    @user1 = User.create!(username: 'user1', email: 'user1@email.com', password: 'password')
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user1)
+  end
+
+  it 'I see information about the movie' do
+    visit discover_path
+
+    VCR.use_cassette('movie_data_details') do
+      fill_in :q, with: 'Phoenix'
+      click_on('Search by Movie Title')
+
+      movie_title = find(".title", match: :first).text
+      vote_average = find(".vote_average", match: :first).text
```

Comment:
This is cool.. Is this capybara?

Reply:
Yes! Lots if research 😅 and me just not thinking lol